core/benchmarks/MapStatusesConvertBenchmark-results.txt

```diff
@@ -2,12 +2,12 @@
 MapStatuses Convert Benchmark
 ================================================================================================
 
-OpenJDK 64-Bit Server VM 17.0.10+7-LTS on Linux 6.5.0-1016-azure
+OpenJDK 64-Bit Server VM 17.0.11+9-LTS on Linux 6.5.0-1018-azure
```

Comment:
The test results of the version `1.0.6` based on `JDK17` are better than the version `1.0.5`, but it seems that the test results based on `JDK21` are slightly worse.